api_pos_integration.md

```diff
@@ -0,0 +1,275 @@
+| `lineItems`    | `LineItem[]`   |         | List of the line items. See [Line Item](#line-item)                                                                                            |
+| `tax`          | `Object`       |         | Map of tax rates (as string encoded decimals) to portion of the price that was taxed with this rate                                            |
+| `taxNet`       | `Object`       |         | Map of tax rates (as string encoded decimals) to portion of the net price that was taxed with this rate                                        |
+| `netPrice`     | `int`          |         | The net price                                                                                                                                  |
+| `price`        | `int`          |         | Total price                                                                                                                                    |
+| `state`        | `PaymentState` |         | The [Status of the payment process](api_checkout.md#payment-state). Only `pending`, `processing`, `successful`, `failed` are used in this case |
+| `loyaltyCard`  | `String`       |         | The loyalty card identifier provided by the user                                                                                               |
+| `processedBy`  | `String`       |         | String identifying the PoS which processes the checkout process                                                                                |
+| `failedReason` | `String`       |         | Message describing the conditions under which the checkout process was marked as failed                                                         |
+
+#### Line Item
+
+| Parameter     | Type     | Default | Description                                                          |
+|---------------|----------|---------|----------------------------------------------------------------------|
+| `scannedCode` | `string` |         | Scanned code                                                         |
+| `amount`      | `int`    |         | Number of products / packages                                        |
+| `weight`      | `int`    | 0       | Weight of product in case of of a weighable (a not packaged) product |
+| `units`       | `int`    | 0       | Number of units in a package in case of bundle or piece product      |
+| `price`       | `int`    |         | Price of the product in case of an encoded price                     |
+| `taxRate`     | `string` |         | Tax rate as string encoded decimal                                   |
+
+#### Example
+```
+{
+  "lineItems" : [
+    {
+      "scannedCode": "0000000000001",
+      "totalPrice" : 798,
+      "amount" : 2,
+      "price" : 399,
+      "taxRate" : 19,
+      "sku" : "1"
+    },
+    {
+      "scannedCode": "0000000000002"
+      "price" : 1099,
+      "totalPrice" : 1099,
```

Comment:
`totalPrice` -> sollte in der Tabelle ergänzt werden

Reply:
Check